core/domain/summary_services.py

```diff
@@ -339,6 +457,21 @@ def _get_displayable_collection_summary_dicts(collection_summaries):
 def get_library_groups(language_codes):
     """Returns a list of groups for the library index page. Each group has a
     header and a list of dicts representing activity summaries.
+
+    Args:
+        language_codes: list(str). A list of language codes. Filter explorations
```

Comment:
A list of language codes. Only explorations with these languages will be returned.

("Filter ..." sounds like an instruction.)

Please make similar changes below.